tests/components/matter/test_climate.py

```diff
@@ -25,6 +25,16 @@ async def thermostat_fixture(
     return await setup_integration_with_node_fixture(hass, "thermostat", matter_client)
 
 
+@pytest.fixture(name="room_airconditioner")
+async def room_airconditioner(
+    hass: HomeAssistant, matter_client: MagicMock
+) -> MatterNode:
+    """Fixture for a thermostat node."""
```

Comment:
nit: "thermostat" in docstring unrelated to thermostat.

Suggestion with spelling copied from other new docstring:
```suggestion
    """Fixture for a Room Airconditioner node."""
```

Or another variant which might be preferred:
```suggestion
    """Fixture for a room air conditioner node."""
```
(other docstring should also be updated then)